runtime/tests/shared/CacheDirPerm.cpp

```diff
@@ -37,8 +36,8 @@ extern "C"
 #define J9SH_DEFAULT_CTRL_ROOT "/tmp"
 
 #define USER_PERMISSION_STR "0700"
-#define USER_PERMISSION		0700
-#define NEW_DIR_PERM	0760
+#define USER_PERMISSION      0700
+#define NEW_DIR_PERM         0760
```

Comment:
We've opted not to try and line text up like this, since it never works properly between the IDE and github. If you need to make this unnecessary change, I suggest just using a single space.

Reply:
Feel free to make use of the 'Hide whitespace changes' in the 'Diff settings' menu.